src/systems/environment-system.js

```diff
@@ -194,3 +201,24 @@ AFRAME.registerComponent("environment-settings", {
     backgroundColor: { type: "color", default: defaultEnvSettings.background }
   }
 });
+
+AFRAME.registerComponent("reflection-probe", {
+  schema: {
+    size: { default: 1 },
+    envMapTexture: { type: "map" }
+  },
+
+  init: function() {
+    this.el.object3D.updateMatrices();
+
+    const box = new THREE.Box3()
+      .setFromCenterAndSize(new THREE.Vector3(), new THREE.Vector3().setScalar(this.data.size * 2))
+      .applyMatrix4(this.el.object3D.matrixWorld);
+
+    this.el.setObject3D("probe", new THREE.ReflectionProbe(box, this.data.envMapTexture));
+
+    this.el.sceneEl.object3D.add(
```

Comment:
Not sure if this is important (as this is mostly for testing purposes) but we are showing the helper even in non scene debug mode and we are not cleaning the helper geo when the component is remove.